x/ccv/provider/keeper/power_shaping.go

```diff
@@ -89,7 +89,7 @@ func (k Keeper) UpdateMinimumPowerInTopN(ctx sdk.Context, consumerId string, old
 
 // CapValidatorSet caps the provided `validators` if chain with `consumerId` is an Opt In chain with a validator-set cap.
 // If cap is `k`, `CapValidatorSet` returns the first `k` validators from `validators` with the highest power.
-func (k Keeper) CapValidatorSet(
+func (k Keeper) CapValidatorSet2(
```

Comment:
Use a better name to express what this does. Also, the function name does not match comment.

Reply:
Is this function used anymore? If not, remove.

Reply:
It's the old function and it's not used anymore, so now I removed it.